### ISSUE TITLE
Builder is broken

### ISSUE THREAD

**Issue by nicolasgere** [open-telemetry/opentelemetry-collector#6476]
**Describe the bug**
The builder is not adding default core modules and they can't be added

**Steps to reproduce**
Just follow the readme of the builder page https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/builder/README.md

**What did you expect to see?**
I expect the build to be created

**What did you see instead?**
"exporter/loggingexporter/v0.64.0: unknown revision exporter/loggingexporter/v0.64.0"

**What version did you use?**
0.6.1 version of the builder

**What config did you use?**
The one on the readme

**Environment**
Linux and Mac have the same result

**Comment by bogdandrutu** [comment]
cc @codeboten this is because of the not yet released split package

**Comment by codeboten** [comment]
@bogdandrutu correct, this is because the example was updated to a package that hasn't been released yet. It will be released this wednesday, so i suggest we leave it as is, since it will be fixed in 2 days.

@nicolasgere the example config should be:

```
exporters:
  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.63.0
  - gomod: go.opentelemetry.io/collector/exporter/loggingexporter v0.63.1

receivers:
  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
    gomod: go.opentelemetry.io/collector v0.63.1

processors:
  - import: go.opentelemetry.io/collector/processor/batchprocessor
    gomod: go.opentelemetry.io/collector v0.63.1
```

**Comment by codeboten** [comment]
This is fixed with the 0.64.0 release.
